.env.staging.example

```
@@ -1,24 +1,37 @@
 # .env.staging.example — copy to .env.staging and replace every change-me-* value
 # before deploying. This file is tracked in git; it contains ONLY placeholder values.
 # Real secrets must never be committed.
+#
+# Used by: docker compose -f docker-compose.staging.yml --env-file .env.staging
 
 # ── Database ─────────────────────────────────────────────────────────────────
 POSTGRES_PASSWORD=change-me-postgres-password
 DATABASE_URL=postgresql://podex:change-me-postgres-password@db:5432/podex
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the POSTGRES_PASSWORD key

(UnorderedKey)